src/frontend/src/controllers/API/queries/api-keys/use-get-api-keys.ts

```diff
@@ -0,0 +1,42 @@
+import { useDarkStore } from "@/stores/darkStore";
+import { useQueryFunctionType } from "@/types/api";
+import { api } from "../../api";
+import { getURL } from "../../helpers/constants";
+import { UseRequestProcessor } from "../../services/request-processor";
+
+export interface IApiKeys {
```

Comment:
could you please change the name for IGetApiKeys?

Reply:
I think IGetApiKeys its not the best name, because this interface reffers to the array of API Keys, inside the api response  
(IApiQueryResponse). i will set a better interface name such as IApiKeysDataArray